src/thunderbird_accounts/asgi.py

```diff
@@ -9,8 +9,21 @@
 
 import os
 
+from django.conf import settings
 from django.core.asgi import get_asgi_application
 
+from servestatic import ServeStaticASGI
+
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'thunderbird_accounts.settings')
 
+
+def immutable_file_test(path, url):
+    # Match vite (rollup)-generated hashes, à la, `some_file-CSliV9zW.js`
+    import re
+    return re.match(r'^.+[.-][0-9a-zA-Z_-]{8,12}\..+$', url)
+
+
 application = get_asgi_application()
+application = ServeStaticASGI(
+    application, root=settings.STATIC_ROOT, prefix='static', immutable_file_test=immutable_file_test
+)
```

Comment:
I see that `immutable_file_test` will cache files generated by rollup. Is there any reason for it not to cache other kinds of static files?

Reply:
It should cover all cases, but it's a requirement if you want to cache files from WhiteNoise/ServeStatic